index.bs

```diff
@@ -469,11 +475,11 @@ aligned(8) class SampleTransform {
 
 The result of any computation underflowing or overflowing the intermediate bit depth is replaced by -2<sup><code>[=sato/num_bits=]</code>-1</sup> and 2<sup><code>[=sato/num_bits=]</code>-1</sup>-1, respectively. Encoder implementations should not create files leading to potential computation underflow or overflow. Decoder implementations shall check for computation underflow or overflow and clamp the results accordingly. Computations with [=sato/operands=] of negative values use the two’s-complement representation.
 
-<dfn noexport for="sato">token_count</dfn> is the expected number of [=sato/tokens=] to read.
+<dfn noexport for="sato">token_count</dfn> is the expected number of [=sato/tokens=] to read. <assert>The value of <code>[=sato/token_count=]</code> shall be greater than 0.</assert>
```

Comment:
This is indirectly covered by the single remaining stack element constraint. I am fine with making it explicit here.

Note: We considered naming it `token_count_minus1` but it seemed inconsistent with other similar boxes.

Reply:
Sounds good. Keeping it explicitly required here